vscode-dbt/snippets/snippets.json

```diff
@@ -45,14 +45,23 @@
         ],
         "description": "Jinja Filter"
     },
-    "For": {
-        "prefix": "for",
+    "For (List)": {
+        "prefix": "for list",
         "body": [
-            "{% for ${1:item} in ${2:sequence} %}",
+            "{% for ${1:item} in ${2:list} %}",
```

Comment:
see my comment about renaming. If you're going to want to add a for that is already pre formatted for lists and only lists, I'd make one that actually helps to this end fully by doing  as shown:
```suggestion
            "{% for ${1:item} in [${2:list}] %}",
```

Reply:
I think it's preferable to leave as it was initially. The cost of typing `[` and `]` is very minimal, so I like it that way as it gives the user flexibility: either you call a predefined list, or you type your own list

Reply:
It's up to you. I think I actually like an additional more specific version and given the type of keyboard I type on I like not to have to hit those if possible. But no pressure.

Reply:
^^ ok you lazy guy, I'll add it just for you!

Reply:
Love it! Also thanks for keeping the whitespace handling consistent with the rest. It's safer that way.